### PR TITLE
Qa backend court info

### PR DESCRIPTION
**Pull request by karthikmsqa**
# Description

This PR includes the following proposed change(s):
- Updated validations for the following endpoints:
- /submission/{id}/generateUrl
- /submission/{id}
- ​/submission​/{id}​/filing-package

## Type of change
- [x] New test (non-breaking change which adds tests)

## How Has This Been Tested?
localhost demo mode

Please describe the tests that you ran to verify your changes.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backend/generateurlpayload/GenerateUrlPayload.java [hunk 1]
```diff
@@ -1,34 +1,130 @@
...
-        Error error = new Error(STRING);
-        Cancel cancel = new Cancel(STRING);
-        Headers headers = new Headers(STRING, STRING, STRING);
-        SubmissionAccess submissionAccess = new SubmissionAccess(STRING,"GET", headers );
-        DocumentProperties documentProperties = new DocumentProperties("DFCL", STRING, submissionAccess);
-        Navigation navigation = new Navigation(success,error,cancel);
+    public String validGenerateUrlPayload() throws IOException {
+        generateUrlRequest = new GenerateUrlRequest();
+
+        TYPEFIRST = STRING;
```
Comment (alexjoybc) [inline tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backend/generateurlpayload/GenerateUrlPayload.java:37]: I would assign a value first

File: tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backend/generateurlpayload/GenerateUrlPayload.java [hunk 2]
```diff
@@ -1,34 +1,130 @@
...
+import ca.bc.gov.open.jag.efilingapi.qa.api.model.*;
 import com.fasterxml.jackson.databind.ObjectMapper;
 
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
 public class GenerateUrlPayload {
+
+    private String DIVISION = null;
```
Comment (alexjoybc) [inline tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backend/generateurlpayload/GenerateUrlPayload.java:12]:
```suggestion
    private static String DIVISION = "division";
```

File: tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backend/generateurlpayload/GenerateUrlPayload.java [hunk 3]
```diff
@@ -1,34 +1,130 @@
...
+    private String LOCATION = null;
+    private String PARTICIPATIONCLASS = null;
+    private String PROPERTYCLASS = null;
+    private String NAME = null;
+    private String DESCRIPTION = null;
+    private String TYPEFIRST = null;
+    private String TYPESECOND = null;
+    private String DISPLAYNAME = null;
+
+    public String SUCCESS_URL = null;
```
Comment (alexjoybc) [inline tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backend/generateurlpayload/GenerateUrlPayload.java:24]:
```suggestion
    public static String SUCCESS_URL = "http://success.com";
```

File: tests/src/test/java/stepDefinitions/backendstepdefinitions/GenerateUrlAndSubmissionTest.java [hunk 4]
```diff
@@ -119,11 +116,51 @@ public void idIsSubmittedWithGetHttpRequest(String resource) throws IOException
...
+
+
+    @Then("verify universal id, user details, account type and identifier values are returned and not empty")
+    public void verifyUniversalIdUserDetailsAccountTypeAndIdentifierValuesAreReturnedAndNotEmpty() throws IOException {
         jsonPath = new JsonPath(response.asString());
 
+        String validExistingCSOGuid = JsonDataReader.getCsoAccountGuid().getValidExistingCSOGuid();
+
+        universalId = jsonPath.get("userDetails.universalId");
         firstName = jsonPath.get("userDetails.firstName");
```
Comment (alexjoybc) [inline tests/src/test/java/stepDefinitions/backendstepdefinitions/GenerateUrlAndSubmissionTest.java:164]: maybe use jackson in the future?

File: tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backend/generateurlpayload/GenerateUrlPayload.java (hunk 1 above)
Comment (karthikmsqa) [reply tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backend/generateurlpayload/GenerateUrlPayload.java:37]: Addressed. Thanks!

File: tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backend/generateurlpayload/GenerateUrlPayload.java (hunk 3 above)
Comment (karthikmsqa) [reply tests/src/main/java/ca/bc/gov/open/jagefilingapi/qa/backend/generateurlpayload/GenerateUrlPayload.java:24]: Addressed. Thanks @alexjoybc .

**Pull request merged**